packages/visx-xychart/src/components/Tooltip.tsx

```diff
@@ -178,11 +178,30 @@ function TooltipInner<Datum extends object>({
   }
 
   // collect positions + styles for glyphs; glyphs always snap to Datum, not event coords
-  const glyphProps: GlyphProps[] = [];
+  const glyphs: React.ReactNode[] = [];
+
+  const renderGlyph =
+    renderGlyphProp ||
+    (<Datum extends object>(props: RenderGlyphProps<Datum>) => {
```

Comment:
might be able to pull this outside the render function as `defaultRenderGlyph` since it should be a constant?

Reply:
Had it in there for now because it accesses the theme for the `stroke`

Reply:
could potentially still move it out and use context to access the theme but nbd